fundingmanager_test.go

```diff
@@ -150,7 +147,8 @@ func createTestFundingManager(t *testing.T, pubKey *btcec.PublicKey,
 	tempTestDir string, hdSeed []byte, netParams *chaincfg.Params,
 	chainNotifier chainntnfs.ChainNotifier, estimator lnwallet.FeeEstimator,
 	sentMessages chan lnwire.Message, sentAnnouncements chan lnwire.Message,
-	publTxChan chan *wire.MsgTx, shutdownChan chan struct{}) (*fundingManager, error) {
+	publTxChan chan *wire.MsgTx, shutdownChan chan struct{},
```

Comment:
At first glance, the amount of parameters passed into this function is a bit excessive. Instead, it seems possible that this function itself can instantiate all the items required to create a test funding manager _within_, and instead return a fully populated "test fixture" for the funding manager itself which can be manipulated as necessary within the tests.

Reply:
Totally agree. Refactored :)